jwst/tests_nightly/general/nircam/out.wfs

```diff
@@ -0,0 +1,203 @@
+
+
+START
```

Comment:
Why are we checking in old `pandokia` log?

Reply:
I'll delete this too